django_mongodb/compiler.py

```diff
@@ -212,8 +231,20 @@ def _get_ordering(self):
             field_ordering.append((opts.get_field(name), ascending))
         return field_ordering
 
+    @cached_property
+    def collection_name(self):
+        return self.query.get_meta().db_table
+
     def get_collection(self):
-        return self.connection.get_collection(self.query.get_meta().db_table)
+        return self.connection.get_collection(self.collection_name)
+
+    def get_lookup_pipeline(self):
+        result = []
+        for alias in tuple(self.query.alias_map):
```

Comment:
why do we need to cast this as a tuple before iterating on it?

Reply:
It was copied from Django, but may be obsolete: https://github.com/django/django/pull/18357

Reply:
There is an answer there, saying that the query.alias_map is modified during the for loops. 🤔 I didn't noticed that, maybe I have to check it.